dca632fffe43df3e.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "dca632fffe43df3e",
```

Comment:
please use uppercase for the ID, and also for the file name (but not the `. json` extension)